packages/playground/package.json

```diff
@@ -54,7 +54,9 @@
     "vscode-languageserver-textdocument": "~1.0.1",
     "vscode-languageserver": "~7.0.0",
     "lzutf8": "~0.6.1",
-    "debounce": "~1.2.1"
+    "debounce": "~1.2.1",
+    "swagger-ui": "~4.12.0",
+    "swagger-ui-react": "~4.13.0"
```

Comment:
I think we don' tneed the swagger-ui-react as it wasn't working with react18 and you reimplemented it